pkg/reconciler/build/build.go

```diff
@@ -61,21 +59,21 @@ func NewController(opt reconciler.Options, k8sClient k8sclient.Interface, inform
 }
 
 type Reconciler struct {
-	Client            versioned.Interface
-	Lister            buildlisters.BuildLister
-	MetadataRetriever MetadataRetriever
-	K8sClient         k8sclient.Interface
-	PodLister         v1Listers.PodLister
-	PodGenerator      PodGenerator
+	Client               versioned.Interface
+	Lister               buildlisters.BuildLister
+	K8sClient            k8sclient.Interface
+	PodLister            v1Listers.PodLister
+	PodGenerator         PodGenerator
+	metadataDecompressor metadataDecompressor
 }
 
-func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
+func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
```

Comment:
For consistency all reconcilers have a `c` receiver. Might make sense to switch them to `r` but, can we do that on its own?